SPECS/python-botocore/python-botocore.spec

```diff
@@ -0,0 +1,1860 @@
+Summary:        Low-level, data-driven core of boto 3
+Name:           python-%{pypi_name}
+# NOTICE - Updating this package requires updating python-boto3
+Version:        1.24.33
+Release:        2%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/boto/botocore
+Source0:        https://github.com/boto/botocore/archive/refs/tags/%{version}.tar.gz#/%{pypi_name}-%{version}.tar.gz
+%global pypi_name botocore
```

Comment:
Generally, this definition should be at the top of the file.

Reply:
Moved all %global to top of file.